src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -237,14 +271,25 @@ public virtual async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAc
                     {
                         AgreementsToSign = pendingAgreements.Count() > 0,
                         Reservations = reservationsResponse.Reservations.ToList(),
+                        CohortsCount = cohortsCount,
+                        ApprenticeshipsCount = apprenticeshipResponse?.ApprenticeshipsCount ?? 0,
+                        NumberOfDraftApprentices = draftApprenticeshipCount,
+                        CourseName = singleDraftApprenticeship.CourseName,

```

Comment:
CourseName, Startdate and EndDate shouldn't be at this level of the viewmodel.  This level is just for counts and bools (as an optimisation) and collections of child models.  There AccountDashboardViewModel should contain a collection of CohortViewModels which each contain a collection of ApprenticeshipViewModels.

Reply:
Done.